app/helpers/react_components/track/credits.rb

```diff
@@ -1,16 +1,15 @@
 module ReactComponents
   module Track
     class Credits < ReactComponent
```

Comment:
At some point we should probably rename this to something more generic, as it doesn't necessarily have to do anything with a track. Maybe it should be `Common::Credits`, which is also how it is named in React.